src/api.ts

```diff
@@ -232,26 +232,32 @@ export async function getMessageStatus(
   });
 
   if (message.aggregatedDeliveryReceipt) {
-    const sdkMessage = getSdkMessageObject(message);
-    const receipts = await sdkMessage.getDetailedDeliveryReceipts(); // paginated backend query every time
```

Comment:
I was referring to this line removal. I just imagined that since it was here there was a reason to fetch status using this method every time, not just obtaining it from existing property.